frontend/components/templates/ConfigurationDetails/ConfigurationDetails.tsx

```diff
@@ -87,7 +87,7 @@ const ConfigurationDetails: React.FC<ConfigurationDetailsProps> = ({
   return (
     <form>
       <AppLayout
-        title={"test"}
+        title={"Configuration"}
```

Comment:
```suggestion
        title="Configuration"
```
I think you might want to prefer to use this formatting (as it's used in other places). Btw, this formatting stuff should be checked by some tool not manually, ex. [Prettier](https://prettier.io). We're using it in other projects ;)